src/components/AnonymousChat/child/ChatItemTargetText.tsx

```diff
@@ -98,14 +130,18 @@ const ChatItemTargetText = ({
                   <View testID="chat-item-user-info" style={styles.chatTitleContainer}>
                     <Text style={[styles.userText, textStyle(false)]}>{username}</Text>
                     <View style={dotStyle(false)} />
-                    <Text style={[styles.timeText, textStyle(false)]}>{time}</Text>
+                    <Text testID="timestamp" style={[styles.timeText, textStyle(false)]}>
+                      {calculateTime(time, true)}
+                    </Text>
                   </View>
                 )}
+
                 <ChatReplyView
                   type={chatType}
                   messageType={messageType}
                   replyData={data?.reply_data}
                 />
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

You have replaced the direct time display with the `calculateTime` function. This is a good change as it improves the user experience by providing a more human-readable timestamp. However, ensure that the `calculateTime` function handles all edge cases and formats the time correctly in all scenarios.


<!-- This is an auto-generated comment by OSS CodeRabbit -->